android-n/ActiveNotifications/ActiveNotificationsFragment.cs

```diff
@@ -122,13 +123,8 @@ public void UpdateNumberOfNotifications ()
 			* for more information.
 			*/
 			try {
-				// Query the currently displayed notifications.
-				StatusBarNotification[] activeNotifications = notificationManager.GetActiveNotifications ();
-
-				int totalNotifications = activeNotifications.Length;
-				numberOfNotifications.Text = GetString (Resource.String.active_notifications, totalNotifications);
-
-				CommonSampleLibrary.Log.Info (TAG, GetString (Resource.String.active_notifications, totalNotifications));
+				numberOfNotifications.Text = GetString (Resource.String.active_notifications, numberOfNotificationsCount);
```

Comment:
I'm not sure I consider this a fix, as the sample is intended to demonstrate the Notifications APIs added in API 23 and later. I don't believe we should be removing the call to NotificationManager.GetActiveNotifications.

Perhaps the original Java source can be used to identify a better approach? https://github.com/googlesamples/android-ActiveNotifications/blob/master/Application/src/main/java/com/example/android/activenotifications/ActiveNotificationsFragment.java

Reply:
Updated with fix from google sample.